snuba/utils/manage_topics.py

```diff
@@ -10,16 +10,18 @@
 logger = logging.getLogger(__name__)
 
 
-def create_topics(client: AdminClient, topics: Sequence[Topic]) -> None:
+def create_topics(
+    client: AdminClient, topics: Sequence[Topic], num_partitions: int = 1
+) -> None:
     topics_to_create = {}
 
     for topic in topics:
         topic_spec = KafkaTopicSpec(topic)
         logger.debug("Adding topic %s to creation list", topic_spec.topic_name)
         topics_to_create[topic_spec.topic_name] = NewTopic(
             topic_spec.topic_name,
-            num_partitions=topic_spec.partitions_number,
-            replication_factor=topic_spec.replication_factor,
+            num_partitions=num_partitions,
+            replication_factor=1,
```

Comment:
is this only used locally?

Reply:
yeah, create_topics will never be used outside a dev environment